jkube-kit/resource/helm/src/test/resources/it/expected/openshift/values.yaml

```diff
@@ -13,9 +13,9 @@
 #
 
 deployment:
-  replicas: "1"
+  replicas: 1
   limits:
-    memory: 512Mi
+    memory: "512Mi"
```

Comment:
Why have you wrapped it in quotes in `expected/openshift/values.yaml` but not in `expected/kubernetes/values.yaml`?

Reply:
For this value it doesn't really make sense because logic says that the value should be deserialized as a String. The goal here is to clearly highlight that this is treated as a String, while other values (such as replicas) can be treated as integers.

This was not the case before, when everything was dealt as a String.
This will become more clear with follow up PRs regarding #2400